tests/test_multi_graph_solvers.py

```diff
@@ -1,187 +0,0 @@
-import sys
```

Comment:
This file should not be deleted

Reply:
I delete this file locally so that I can speed up my test.